### ISSUE TITLE
make data accessible via console

### ISSUE THREAD

**Issue by erikdstock** [thirdcreed/Nevada#18]
perhaps something in app.js like
```js
React.useEffect(() => {
  window.getResults = () => data
  window.getCsv = () => ...
  window.copyResults = () => copy(data)
}
```
etc
Also consider logging a message to the console and adding copy somewhere on the site to let users know about this

**Comment by thirdcreed** [comment]
also server paths to the csv files themselves.

**Comment by erikdstock** [comment]
@thirdcreed , do you think we should have a timestamp property added by `massageResults` (if not on the generated csv as well)?

**Comment by thirdcreed** [comment]
That's a good idea.  I can do that on the web server. It's in the filenames.  I was actually planning on making a route that grabbed past files as well.  So people can time travel.

**Comment by erikdstock** [comment]
closed by #21
